### ISSUE TITLE
Inline Code & Main Page Headings Unreadable + Book Navigation issue

### ISSUE THREAD

**Issue by TechAsh** [domainzero/bookstack-dark-theme#16]
Hi,
I've just found your theme and wanted to try it out. I've found a few issues which I've listed below. Unfortunately I'm no good with CSS, so I don't know how to fix it.
- Inline Code is being displayed as white text in a white box.
- The headings "My Recently Viewed" & "Recent Activity" on the main page are dark grey on a grey background.
- The coloured vertical lines in the "Book Navigation" sidebar seem duplicated or misaligned.

Theme Version: 36b5ab65e830b5934d0542c0e67d622b78fb207e
Bookstack Version: 0.27.5

Thanks :smiley:

**Comment by domainzero** [comment]
Hey @TechAsh 
Do you have any screenshots you can show me?
I'm not experiencing either of those issues. There is a block of CSS at the bottom of the stylesheet (that I should admittedly move to the top) that you'll need to add to the Custom HTML Head Content section of your Bookstack settings.

Here's that block, complete with style tags.

```css
<style>
code{white-space:wrap}
.page-content del {background: #FF5555!important; }
.card .entity-list-item:not(.no-hover):hover {background-color: #44475a !important;}
.page-content ins {background: #50fa7b!important;color: #282a36!important;}
.suggestion-box {background-color: #282a36!important;border: 1px solid #BBB;box-shadow: none!important;border-radius: 0!important; }
.suggestion-box li:last-child {border-bottom: 0; }
.suggestion-box li.active {background-color: #44475a!important; }
.card .entity-list-item:not(.no-hover):hover {background-color: #44475a!important; }
.card-title {text-transform: uppercase; color: #f8f8f2; fill: #f8f8f2; font-weight: 400;}
.markdown-editor-display{background-color: #282a36; }
.markdown-editor-display body {background-color: #282a36; }
.breadcrumb-listing-entity-list {background-color:#282a36 !important;}
.breadcrumb-listing .breadcrumb-listing-toggle:hover {fill: #f8f8f2;border-color: transparent;}
.entity-meta {fill:#f8f8f2!important;}
.book-tree .sidebar-page-list .entity-list-item.selected {background-color:#44475a;}
</style>
```

**Comment by TechAsh** [comment]
Ah, there is no mention of that step in the installation instructions.
Adding that extra CSS to the Custom Head HTML fixed the headings on the main page. However the other 2 issues are still there. Screenshot attached.
![Screenshot_2020-01-16](https://user-images.githubusercontent.com/16271831/72548794-5a99f100-3887-11ea-8e4a-5f7ad3b3deb1.png)
(I also have the Dracula codemirror theme installed)

**Comment by skj-dev** [comment]
FWIW, I replaced the `code` section in the `<style>` block from the bottom of the CSS file with the entries that are in the `/* one offs */` section. So instead of

`code {white-space:wrap;}`

I have 

`code {color:#50fa7b;background-color:#282b36;white-space:wrap;}`

and now the code blocks show up nicely. 

<img width="416" alt="Screen Shot 2020-01-19 at 1 27 24 PM" src="https://user-images.githubusercontent.com/688146/72687015-b9e24600-3abf-11ea-9840-87eb25413eee.png">

And I agree that it's not clear that the block at the bottom of the CSS file is commented out, and needs to be put into the `Custom HTML Head Content` section. I skimmed through the CSS before adding it, and completely missed it. 

Having said all of that, thanks for setting this up, it's _real_ nice!

**Comment by domainzero** [comment]
Appreciate the feedback, guys.
Implemented the code fix, as well as fixes for the duplicate left border problem referenced by @TechAsh 
Also updated the README to make it more clear that you need to include some CSS in your Custom HTML Head Content.

Should be fixed in commit dd14904

**Comment by skj-dev** [comment]
Just loaded it up, and it looks great to me! Thanks!

**Comment by davelevine** [comment]
Hi @domainzero ,

I love this theme, but wanted to point something out that I found when resizing the browser. It looks like the menu doesn't render properly (see attached screenshot). This is in version 0.27.5.

I just implemented the changes from commit [dd14904](https://github.com/domainzero/bookstack-dark-theme/commit/dd149046732ec69be40383a929d6bbcdd1565629), although I'm not sure if this was rendering this way prior to this commit.

![Screenshot_2020-01-19_15-35-47](https://user-images.githubusercontent.com/49622406/72688070-eacb7680-3ad1-11ea-99f9-0dffdeb9931a.png)

If you'd rather I create a separate issue for this, please let me know. Also let me know if any additional information is needed.

Thanks!

**Comment by domainzero** [comment]
Looks like your viewport is small enough that it's triggering the mobile CSS rules. 
Add the following lines to your Custom HTML Head Content:
```css
header .header-links {background-color: #282a36;}
header .links a:hover, header .dropdown-container ul li a:hover {background-color:#44475a;}
```

Fixed in 57b37a7 @dlevine87

**Comment by davelevine** [comment]
Worked perfectly! Thanks for doing that so fast! @domainzero

**Comment by domainzero** [comment]
That was a pretty quick fix, and as such it had a slightly unintended consequence :P
In the instance you aren't using the main background color as your application color, you might get some wacky color stuff in your toolbar.

Add the following line to Custom HTML Head Content to make it so the header change only applies when your viewport is small.
```css
@media screen and (max-width: 1000px) {header .header-links {background-color: #282a36;}}
```

This is all fixed in commit 21804f5

**Comment by davelevine** [comment]
I actually didn't realize the unintended consequence until I logged in on my laptop.

Not sure if it's just me, but adding the line above didn't immediate fix the application color. I ended up having to remove the entire block from step 3 of the installation instructions, then reapply it to take affect. This could all just be related to caching, but wanted to mention it in case anyone else has the same issue.

Thanks for all your help, @domainzero!

**Comment by TechAsh** [comment]
Excellent! Thanks. I've just tested the latest version and all the problems I mentioned are fixed :)
I tried testing all the other text styles I could find and did find a few other problems:
- Blockquotes are affected by the same problem as inline-code was. (white on white)
- There is no real way to identify links in text until you hover over them with the mouse.
- The colour picker for text and background colour in the editor is a load of dark squares. (See image)
![Screenshot_2020-01-26 Editing Page Test BookStack](https://user-images.githubusercontent.com/16271831/73139808-d35c3280-4069-11ea-978e-630a4cf5e1ed.png)
Thanks again. I really like the theme.
